### ISSUE TITLE
Slimbot support commands ?

### ISSUE THREAD

**Issue by phuongnguyenkim** [edisonchee/slimbot#24]
Sorry my english not well.
Does slimbot support commands feature ?
I can't find that in slimbot documents.

**Comment by edisonchee** [comment]
Hi @phuongnguyenkim, Slimbot can respond to commands as well. You need to first set up commands for your bot with @BotFather using `/setcommand`. Once that is done, just listen for it with your bot. Do you need an example for this?

**Comment by lgg** [comment]
@edisonchee please, can you clarify: I set command via `/setcommand` in @Botfather

How I can check command calls in bot? For now I parse command
```
slimbot.on('message', message => {
  const currentCommand = commandManager.parseCommand(message.text);
switch (currentCommand) {
        case 'help':
            console.log('help')
            bot.slimbot.sendMessage(currentChatId, 'Help');
            break;
}
});

commandManager.parseCommand = (text) => {
    if (text.charAt(0) === '/') {
        if(text.indexOf(' ') === -1){
            return text.substr(1, text.length);
        }else{
            return text.substr(1, text.indexOf(' '));
        }
    } else {
        return false;
    }
};
```

**Comment by KnorpelSenf** [comment]
Have a look at https://core.telegram.org/bots/api#message and see what kind of information you can find about bot commands in the field entities. That makes parsing a lot easier.
Remember that a command can be of the form /help or /help@botname

**Comment by edisonchee** [comment]
@KnorpelSenf Thanks for helping to answer this!

@phuongnguyenkim @lgg If your questions are answered perhaps you can help to close the thread, or maybe open another issue. :)
